db_setup/extract_seed_data.py

```diff
@@ -0,0 +1,122 @@
+"""This script extracts data required for the plant & location tables in the database."""
+
+
+import requests
+import pandas as pd
+
+
+BASE_URL = "https://data-eng-plants-api.herokuapp.com/plants/"
+LOCATION_LIST_LEN = 5
+LAT_INDEX = 0
+LONG_INDEX = 1
+TOWN_INDEX = 2
+COCO_INDEX = 3
+CONTINENT_CITY_INDEX = 4
+PLANT_RANGE = 51
+GERTRUDE_ID = 1
+CARL_ID = 2
+ELIZA_ID = 3
+
+
+def get_plant_data(plant_id: int) -> list[dict]:
+    """Gets plant data from API using ID."""
+    return requests.get(BASE_URL + str(plant_id), timeout=100).json()
+
+
+def parse_location(location_info: list) -> dict:
+    """Turns a list with location info into a dictionary."""
+    location = {}
+    if location_info:
+        if len(location_info) == LOCATION_LIST_LEN:
+            location["latitude"] = location_info[LAT_INDEX]
```

Comment:
could you combine these if statements? 
if location_info and len(location_info) == LOCATION_LIST_LEN: